src/main/java/org/opentripplanner/routing/impl/DelegatingTransitAlertServiceImpl.java

```diff
@@ -54,10 +56,13 @@ public TransitAlert getAlertById(String id) {
   }
 
   @Override
-  public Collection<TransitAlert> getStopAlerts(FeedScopedId stop) {
+  public Collection<TransitAlert> getStopAlerts(
```

Comment:
Can you add some javadoc on the class here (not needed for the methods) so it's easier to understand what is the purpose of this Delegating service.

Reply:
Done